text/0517-io-os-reform.md

```diff
@@ -1155,7 +1155,174 @@ RFC recommends they remain unstable.
 #### `stdin`, `stdout`, `stderr`
 [stdin, stdout, stderr]: #stdin-stdout-stderr
 
-> To be added in a follow-up PR.
+The current `stdio` module will be removed in favor of these constructors in the
+`io` module:
+
+```rust
+pub fn stdin() -> Stdin;
+pub fn stdout() -> Stdout;
+pub fn stderr() -> Stderr;
+```
+
+* `stdin` - returns a handle to a **globally shared** standard input of
+  the process which is buffered as well. All operations on this handle will
```

Comment:
There are operations other than `lock` on `Stdin`, meaning a user can do things with it without locking. (I'm now inclined to believe I just misread: and this sentence is saying those operations would just call `lock` internally.)